content/applications/websites/ecommerce/customer_accounts.rst

```diff
@@ -2,42 +2,128 @@
+- :guilabel:`Free sign up`: Every website visitor can create an account and sign in. They will
+  get access to the :doc:`portal <../../general/users/portal>` by default.
+
+If account creation is restricted to invited customers only, go to :menuselection:`Website -->
+eCommerce --> Customers`, switch to the :guilabel:`List` view, and select the customers who
+should gain access to the :doc:`portal <../../general/users/portal>`.
+Click the :icon:`fa-cog` :guilabel:`Actions` button, then :guilabel:`Grant portal access`.
+In the :guilabel:`Portal Access Management`, review the selected customers and click
+:guilabel:`Grant Access` to confirm.
+
+Once done, the relevant customers receive an email confirming their account creation, including
+instructions on setting a password and activating their account.
 
 .. note::
-   Customers can only have an account if the
-   :ref:`sign in/up at checkout <ecommerce/checkout/policy>` option allows for account creation.
+   - When selecting the :guilabel:`Free sign up`, a clickable :guilabel:`Don't have an account?`
+     link appears under the login form on the website.
+   - The :guilabel:`On invitation` option is especially useful for B2B businesses that prefer
+     to keep prices hidden on the website. Only customers who have been granted portal access are
+     able to view their pricing.
 
 .. tip::
-   Similarly to the rest of the website, the customer account page can be customized with **content
-   blocks** and other features through the **website builder**.
-
-.. seealso::
-   :doc:`/applications/general/users/portal`
+   It is possible to configure a website form with a :guilabel:`Create a Customer` :ref:`action
+   <website/building_blocks/form>` to automatically create a customer record in the backend when
+   filled in.
 
 Access restriction
 ==================
 
-It is possible to allow or restrict the documents to which customers have access through the website
-builder. Log in your **own** account with your Odoo database credentials, and go to
-:menuselection:`Edit --> Customize`. From the website builder menu, enable or disable the documents
-customers can have access to.
+Once a customer account is created, it is still possible to adjust the access rights
+either globally or for individual users:
+
+- :ref:`Revoke access or re-invite a customer <portal/access>`
+  using the related buttons in the :guilabel:`Portal Access Management` pop-up.
+- Restrict :ref:`access to the shop <ecommerce/customer_accounts/shop-access>`;
+- Decide whether customers need to create an account to :ref:`complete the checkout
+  <ecommerce/customer_accounts/checkout-access>`.
+
+.. note::
+   Users can only have one :doc:`portal access <../../general/users/portal>` per email.
+
+.. tip::
+   It is also possible to define the types of documents customers have access to. To do so, open
+   the :doc:`website builder <../website/web_design>`, then enable or disable access to specific
+   documents as needed.
```

Comment:
```suggestion
   It is also possible to define the types of documents customers have access to. To do so, open
   the customer portal page (e.g., `https://www.example.com/my/home`) and open
   the :doc:`website builder <../website/web_design>`. On the :guilabel:`Customize` tab, enable or disable access to specific documents as needed.
```
I would mention the portal's default location (/my/home), as I don't feel it would be obvious for the average user?